### ISSUE TITLE
Text logging only works on LCD blocks

### ISSUE THREAD

**Issue by ShuRugal** [ManuelSchmitz/SCAM#23]
Greetings!  Not sure if this is the *best* place for this, but...
I could like the ability to send the log output to the seat of a control station, similar to how it can be done with AutoLCD where you tag the station, then add "@<n> AutoLCD" to set it to display on the selected display number.

Any chance of getting that?

**Comment by Aimonxxl** [comment]
This is not the goal of the project. You have a station that receives drones and drones that interact with the station. Just create another script that will send the necessary data to other grids.

**Comment by ShuRugal** [comment]
> This is not the goal of the project. You have a station that receives drones and drones that interact with the station. Just create another script that will send the necessary data to other grids.

I believe that you misunderstood my request.  I probably failed to be clear enough.

What i want is for the dispatcher log to be able to log to displays on control stations, rather than on an LCD.  I like the dispatcher logs to be visible from the location where I set up the scam-gui-controller station, but having it on a full-block LCD is very obtrusive.  I tried command:add-logger:scam-gui-controller, but that did not work.  Presumably because the API handling of control-station LCDs is different from LCD blocks, and the SCAM script is written only to support LCD blocks.
